File: Ohtap E Commerce Domain Models/Development/NetCore/Models Objects/Models Objects/CurrencyDetails/ICurrencyDetailData.cs
using Ohtap.ECommerce.Domain.Models.Objects.Datasets;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.CurrencyDetails
{
    public interface ICurrencyDetailData : CoreObject.IObjectID, CoreObject.IObjectCode, CoreObject.IObjectName, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState
    {
        IDataItemList<IDataItemData> Dataset { get; set; }
    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Objects/CurrencyDetails/ICurrencyDetailData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\CurrencyDetail;


use  Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface ICurrencyDetailData extends CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{

}
